### PR TITLE
DOC: Version 0.0.1 update

### PR DESCRIPTION
**Pull request by daikitag**
Version 0.0.1 release

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -22,10 +22,11 @@ html:
   use_repository_button: true
   use_edit_page_button: true
 
-  # NOTE this gets replaced by the build infrastructure during deployment
-  extra_navbar: __TSTRAIT_VERSION__
-  extra_footer: __TSTRAIT_VERSION__
-
+  extra_footer: |
+    <div>
+    msprime __TSTRAIT_VERSION__
```
Comment (benjeffery) [inline docs/_config.yml:27]:
```suggestion
    tstrait __TSTRAIT_VERSION__
```

**Review by benjeffery — COMMENTED** [review]
Looks good, one place where msprime is mentioned.

File: docs/_config.yml (hunk 1 above)
Comment (daikitag) [reply docs/_config.yml:27]: Thanks for your suggestion! I missed it when I made the commitment.

**Comment by daikitag** [comment]
I'm sorry, but does anyone know why this error is being caused? It seems that the packages listed in requirements are not installed before building the documentation.

**Comment by benjeffery** [comment]
Try bumping the version number here: https://github.com/tskit-dev/tstrait/blob/main/.github/workflows/docs.yml#L38

**Comment by daikitag** [comment]
I recreated the cache in Github by specifying a new key, and the test is working perfectly fine now. I imagine that there was an unknown error with the previous key.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
+
+- Initial stable release of tstrait in PyPI https://pypi.org/project/tstrait/
+- Initial stable release of tstrait in conda-forge 
+
+### Documentation:
+
+- tstrait description in https://tskit.dev/software/
+
+### Contributors:
+
```
Comment (jeromekelleher) [inline CHANGELOG.md:15]: Missing Gertjan Bisschop here

**Review by jeromekelleher — COMMENTED** [review]
LGTM, one fix.

File: CHANGELOG.md (hunk 2 above)
Comment (daikitag) [reply CHANGELOG.md:15]: Thank you for letting me know about it. I just included the people who made the commitment during the period, but from next time, I will definitely include everyone's name, as I wouldn't have been able to complete these commitments without everyone's support.

**Comment by benjeffery** [comment]
@mergify rebase

**Comment by benjeffery** [comment]
@mergifyio rebase

**Pull request merged**